code/modules/martial_arts/combos/sleeping_carp/gnashing_teeth.dm

```diff
@@ -0,0 +1,18 @@
+/datum/martial_combo/sleeping_carp/gnashing_teeth
+	name = "Gnashing Teeth"
+	steps = list(MARTIAL_COMBO_STEP_HARM, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "Every second consecutive punch deals extra damage, and you will shout to strike fear into your opponent's heart."
+
+/datum/martial_combo/sleeping_carp/gnashing_teeth/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	user.do_attack_animation(target, ATTACK_EFFECT_PUNCH)
+	var/atk_verb = pick("precisely kick", "brutally chop", "cleanly hit", "viciously slam")
+	target.visible_message("<span class='danger'>[user] [atk_verb]s [target]!</span>",
+					"<span class='userdanger'>[user] [atk_verb]s you!</span>")
+	playsound(get_turf(target), 'sound/weapons/punch1.ogg', 35, TRUE, -1)
+	add_attack_logs(user, target, "Melee attacked with martial-art [MA] :  Gnashing Teeth", ATKLOG_ALL)
+	target.apply_damage(20, BRUTE)
+	if(target.health >= 0)
+		user.say(pick("HUAH!", "HYA!", "CHOO!", "WUO!", "KYA!", "HUH!", "HIYOH!", "CARP STRIKE!", "CARP BITE!"))
+	else
+		user.say(pick("BANZAIII!", "KIYAAAA!", "OMAE WA MOU SHINDEIRU!", "YOU CAN'T SEE ME!", "MY TIME IS NOW!", "COWABUNGA")) // COWABUNGA
```

Comment:
Why the health threshold?

Reply:
if they are in crit they will shout the phrases that used to be shouted when elbow drop happened. so those are the "finisher" phrases